src/index.js

```diff
@@ -0,0 +1,178 @@
+let temperatureLabel = null;
+let temperatureIncreaseButton = null;
+let temperatureDecreaseButton = null;
+let getRealtimeTemperatureButton = null;
+let skyImage = null;
+let landscapeImage = null;
+let cityNameInput = null;
+let cityNameResetButton = null;
+let skySelectValue = null;
+let gardenBackground = null;
+let temperatureValue = null;
+let defaultCityName = null;
+
+const showTemp = () => {
+    temperatureLabel.textContent = temperatureValue;
+}; 
+
+const showCity = () => {
+    cityNameLabel.textContent = defaultCityName;
+    cityNameInput.value = defaultCityName;
+}; 
+
+const setTempLabelColor = (temperature) => {
```

Comment:
The `setAttribute` calls don't return anything, so prefer to leave off the `return` statements. They give the impression that the function's focus is on returning a value, when it's the `setAttribute` calls on their own that are important.